tests/Feature/ApplicationContainerPresenceTest.php

```
@@ -0,0 +1,54 @@
+    $heading = file_get_contents(resource_path('views/livewire/project/application/heading.blade.php'));
+    $mobileActions = str($heading)
+        ->after('id="application-mobile-actions"')
+        ->before('<div class="hidden" aria-hidden="true">')
+        ->toString();
+
+    expect($heading)->toContain('<x-application.restart-limit-warning :application="$application" />')
+        ->and($heading)->toContain('$application->container_present === true')
+        ->and($heading)->toContain("\$application->stoppedAfterRestartLimit() ? 'Retry deployment' : 'Deploy'")
+        ->and($heading)->toContain("\$application->stoppedAfterRestartLimit() ? 'Retry deployment (without cache)' : 'Deploy (without cache)'")
+        ->and($heading)->toContain('Remove container')
+        ->and(strrpos($mobileActions, 'Remove container'))->toBeGreaterThan(strrpos($mobileActions, 'Deploy (without cache)'));
```

PHPStan (2.2.8)

[warning] Parameter `#1` $expected of method Pest\Mixins\Expectation<int|false>::toBeGreaterThan() expects DateTimeInterface|float|int|string, int|false given.


(argument.type)